src/ChainSafe.GamingSDK.EVM.WebPageWallet/WebPageWallet.cs

```diff
@@ -30,9 +36,18 @@ public class WebPageWallet : ISigner, ITransactionExecutor
             this.configuration = configuration;
         }
 
+        public delegate string ConnectMessageBuildDelegate(DateTime expirationTime);
+
+        public async ValueTask WillStartAsync()
+        {
+            address = await GetAccountVerifyUserOwns();
+        }
+
+        public ValueTask WillStopAsync() => new(Task.CompletedTask);
+
         public Task<string> GetAddress()
         {
-            throw new NotImplementedException();
+            return Task.FromResult(address!);
```

Comment:
Let's check for a null address here and throw instead of assuming it's initialized. Also, This a very good candidate for and expression-bodied method XD